exercises/practice/bank-account/.docs/instructions.append.md

```diff
@@ -1,5 +1,22 @@
 # Instructions append
 
+````exercim/note
+Python doesn't support "true" concurrency due to the [Global Interpreter Lock][GIL].
+While work is ongoing to create support for [free-threading in Python][free-threading], it is still experimental.
+Current standard library solutions such as [multiprocessing][multiprocessing-module] and [threading][threading-module]  are difficult to implement with the current track tooling.  
```

Comment:
```suggestion
Current standard library solutions such as [multiprocessing][multiprocessing-module] and [threading][threading-module] are difficult to implement with the current track tooling.  
```

Double space here. Double blank line. Otherwise, this looks good to me!